index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="author" content="Dagmawi Yohannes" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>portfolio</title>
+        <meta name="title" content="mobile first" />
+        <meta name="description" content="mobile first web design" />
+        <link rel="icon" type="image/png" sizes="32x32" href="images/logo.png" />
+        <link rel="stylesheet" href="css/styles.css" />
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+        <style>
+          li {
+              display: inline;
+              padding: 20px;
+          }
+        </style>
+    </head>
+    <body>
+        <section class="main">
+          <div>
+            <!-- navbar setup -->
+            <header>
+                <nav>
+                    <ul class="menu">
+                      <li>Portfolio</li>
+                      <li>About</li>
+                      <li>Contact</li>
+                    </ul>
+                    <div class="button">
+                      <a href="">Welcome</a>
+                      <div>
+                        <div class="bar"></div>
+                        <div class="bar"></div>
+                        <div class="bar"></div>
+                      </div>
+                    </div>
+                    <img class="navbar__hamb" src="images/hamburguer.png" alt="hamburguer icon" />
+                  </nav>
+            </header>
```

Comment:
- [ ] Kindly fix the header styling and make it look like the figma template.
### Your design
![image](https://user-images.githubusercontent.com/94551484/219790743-b1a1daab-eca9-43e0-bfd3-e181ff69ef0d.png)
### Figma template
![image](https://user-images.githubusercontent.com/94551484/219790873-e2017751-2143-40d8-b8cc-af776e9a5f5e.png)

Reply:
Done